JavaAssignment/src/vrushali/TimeConversion.java

```diff
@@ -0,0 +1,23 @@
+package vrushali;
+
+public class TimeConversion {
+	
+	static int sec, hour, minutes;
```

Comment:
If you want to use this variable in multiple methods then only create instance variable otherwise it is always advisable to create local variable. 

Rest code looks good to me.

Reply:
Changes are made as per review comments. Removed instance variable and created local variable.